Src/Sarif.PatternMatcher/SearchSkimmer.cs

```diff
@@ -616,8 +616,8 @@ private void RunMatchExpressionForSingleLineRegexes(AnalyzeContext context, Matc
             if (_validators != null && matchExpression.IsValidatorEnabled)
             {
                 string filePath = context.TargetUri.GetFilePath();
-
-                IEnumerable<ValidationResult> validationResults = _validators.Validate(reportingDescriptor.Name,
+                string ruleName = matchExpression.Name ?? reportingDescriptor.Name;
```

Comment:
>ruleName [](http://example.com/codeflow?start=23&length=8)

It turns out that we're flowing the wrong identifiers around for a specific kind of utilization of our extensibility model (which is flexible). I have fixed this here but because it is currently an 'offroad' model and we are urgently driving some other work, I did not stop and create the unit tests this warrants. That will be done in a follow on change to a repo that includes this one as a submodule.

Reply:
Can you add a TODO comment to track the follow up work is completed?

Reply:
Created a work item for this and sent you a message. I don't want to add a TODO comment here because the code is presumed correct. :) The tests will actually be created in a separate workflow.